src/lib/contentful.js

```diff
@@ -0,0 +1,24 @@
+import { createClient } from 'contentful';
+import { environment } from './environments/environment.staging';
```

Comment:
Se tiene que pedir solamente el archivo environment.
https://stackoverflow.com/questions/50313296/how-to-add-set-environment-angular-6-angular-json-file
```suggestion
import { environment } from './environments/environment';
```

Reply:
OK, configurado angular.json